systemd/systemd.go

```diff
@@ -791,7 +827,7 @@ func (s *systemd) Status(unitNames ...string) ([]*UnitStatus, error) {
 	var extendedUnits []string
 
 	for _, name := range unitNames {
-		if strings.HasSuffix(name, ".timer") || strings.HasSuffix(name, ".socket") {
+		if strings.HasSuffix(name, ".timer") || strings.HasSuffix(name, ".socket") || strings.HasSuffix(name, ".target") {
```

Comment:
Although related, this change could live in a separate commit  (or even a separate PR, if it can make sense to have it on its own), because then in that commit's description wou would explain why it's needed.